test/controllers/observations/images_controller_test.rb

```diff
@@ -138,7 +138,7 @@ def test_update_image
       image = images(:agaricus_campestris_image)
       obs = image.observations.first
       assert(obs)
-      assert(obs.rss_log.nil?)
+      assert_not_nil(obs.rss_log)
```

Comment:
... because `obs.rss_log` ain't nil anymore.
All obs fixtures now have (and must have) a corresponding rss_log fixture.